sympy/solvers/tests/test_ode.py

```diff
@@ -2977,16 +2971,6 @@ def test_2nd_power_series_ordinary():
     # FIXME: Maybe there should be a way to check series solutions
     # checkodesol doesn't work with them.
     C1, C2 = symbols("C1 C2")
-    eq = f(x).diff(x, 2) - x*f(x)
-    assert classify_ode(eq) == ('2nd_power_series_ordinary',)
-    assert dsolve(eq) == Eq(f(x),
-        C2*(x**3/6 + 1) + C1*x*(x**3/12 + 1) + O(x**6))
-    assert dsolve(eq, x0=-2) == Eq(f(x),
-        C2*((x + 2)**4/6 + (x + 2)**3/6 - (x + 2)**2 + 1)
-        + C1*(x + (x + 2)**4/12 - (x + 2)**3/3 + S(2))
-        + O(x**6))
-    assert dsolve(eq, n=2) == Eq(f(x), C2*x + C1 + O(x**2))
```

Comment:
Can you add these tests back and change it so that you pass `hint='2nd_power_series_ordinary'` to `dsolve`?